src/views/ContextMenuWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+/*!
+ * © 2024 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ContextMenu, ContextMenuProps } from '@manuscripts/style-guide'
+import React from 'react'
+
+export const ContextMenuWrapper: React.FC<ContextMenuProps> = (props) => {
+  return <ContextMenu {...props} />
```

Comment:
It looks to me like that component doesn't do much.

Reply:
It is removed. ContextMenu is called directly from style-guide...